src/stun_listener.erl

```diff
@@ -212,11 +212,32 @@ udp_recv(Socket, Opts) ->
             erlang:error(Reason)
     end.
 
-format_listener_error(IP, Port, Transport, Opts, Err) ->
+format_listener_error(IP, MinPort, MaxPort, Transport, Opts, Err) ->
     ?LOG_ERROR("Cannot start listener:~n"
                "** IP: ~s~n"
-               "** Port: ~B~n"
+               "** PortRage: ~B-~B~n"
                "** Transport: ~s~n"
                "** Options: ~p~n"
                "** Reason: ~p",
-               [stun_logger:encode_addr(IP), Port, Transport, Opts, Err]).
+               [stun_logger:encode_addr(IP), MinPort, MaxPort, Transport, Opts, Err]).
+
+open_socket(MinPort, MaxPort, OpenSockFunc) ->
+    Count = MaxPort - MinPort,
+    Next = MinPort + stun:rand_uniform(Count + 1) - 1,
```

Comment:
This probably can result in `-1`

Reply:
`stun:rand_uniform(N)` return positive integer not bigger than `N`